lib/base/application.cpp

```diff
@@ -171,7 +171,7 @@ void Application::SetResourceLimits(void)
 	rlimit rl;
 
 #	ifdef RLIMIT_NOFILE
-	rl.rlim_cur = 16 * 1024;
+	rl.rlim_cur = 64 * 1024;
```

Comment:
The better fix would be to move this out of the Application class and back into the Init script or some post-compile-time configuration variable. The reason it was removed was because debian switched to dash which did not provide ulimit -u. Alternatives to debian users are to use ulimit -p or change shbang to bash on init scripts. https://dev.icinga.org/issues/5260